packages/app/components/profile/index.tsx

```diff
@@ -153,17 +150,30 @@ const Profile = ({ address }: { address: string | null }) => {
             animationHeaderPosition={animationHeaderPosition}
             animationHeaderHeight={animationHeaderHeight}
             isBlocked={isBlocked}
+            profileData={profileData?.data}
+            isLoading={isLoading}
+            isError={isError}
           />
         </View>
       </View>
     );
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [headerBgLeft, headerShadow, headerHeight, address, isBlocked]);
+  }, [
+    headerBgLeft,
+    headerShadow,
+    headerHeight,
+    address,
+    animationHeaderPosition,
+    animationHeaderHeight,
+    isBlocked,
+    profileData?.data,
+    isLoading,
+    isError,
+  ]);
 
   const routes = useMemo(
     () =>
       data?.tabs?.map((item, index) => ({
-        title: item?.name,
+        title: item?.name?.replace(/^\S/, (s) => s.toUpperCase()),
```

Comment:
we can use textTransform style with capitalize

Reply:
oh, I remembered, at the time I was considering the situation is that sometimes you need to capitalize the first sentence, just like `This today` instead of `This Today`, via CSS it's not good, so I use `JS` to do this!


figma design snapshot: 

![image](https://user-images.githubusercontent.com/37520667/173656976-3c6f172d-5725-47a7-8929-f4109b942673.png)

Reply:
makes sense 👍